plugins/woocommerce/e2e/tests/activate-and-setup/complete-onboarding-wizard.spec.js

```diff
@@ -1,5 +1,10 @@
 const { test, expect } = require( '@playwright/test' );
 
+const adminEmail =
+	process.env.USE_WP_ENV === '1'
+		? 'wordpress@example.com'
+		: 'admin@woocommercecoree2etestsuite.com';
```

Comment:
Seems like a sensible solution to get us started here.  Cleaning up test data is a follow-up task I'll take care of (soon, hopefully!)